src/clarity-angular/forms/_select.clarity.scss

```diff
@@ -80,11 +80,11 @@
     }
 
     //Multiple Select
-    select[multiple] {
+    select[multiple], select[size] {
         $clr-multiple-select-background: $clr-white;
         $clr-multiple-select-border-color: $gray-light-midtone;
 
-        padding: $clr-baselineRem_0_25;
+        padding: 0;
```

Comment:
may be just add a left and right padding and remove the top and bottom ones? You should check with @reddolan